boa/tui/tui.py

```diff
@@ -30,6 +32,14 @@
 from rich.syntax import Syntax
 from rich.rule import Rule
 
+yaml = YAML(typ="rt")
+yaml.preserve_quotes = True
+yaml.default_flow_style = False
+yaml.indent(sequence=4, offset=2)
+yaml.width = 1000
+# yaml.Representer = ruamel.yaml.representer.RoundTripRepresenter
+# yaml.Loader = ruamel.yaml.RoundTripLoader
```

Comment:
These 2 settings lead to failure in opening interactive mode for me. And for some reasons, errors are not reported on my machine (just like inotify didn't give me any error before)